### DISCUSSION TITLE
Right place to discuss ideas for mission chains, story lore etc? Becuase I have OPINIONS!...

### DISCUSSION THREAD

**Discussion by Vemenous-Repentile** [endless-sky/endless-sky#10113] [Show and tell]
...they're likely poorly informed and of little interest to anyone, but I thought I'd scout out the right place to share them nonetheless.

Apologies if this isn't it. 

****PLOT SPOILERS AHEAD*****

First thing I ought to say is that I've really enjoyed the way the world of Endless Sky has been fleshed and populated with characters, storylines and curious little mysteries. Only started playing at the start of this year and I wish I'd stumbled on it sooner. 

Second thing I ought to say is that I have precisely 0 coding skills on a scale of 0-1, meaning that I'm self-consciously well-aware that I'm most likely far less useful than anyone who understands how to make computer-games computer-game. The previous statement was essentially superfluous, because I really just wanted to say thank you to all the technically minded people who made a game that I've spent ages playing, for free. 

Third thing, for anyone who's been brave/bored enough to read this far, is that I had a few ideas which I thought might be interesting as ways to expand some of the plugins I've been enjoying or the "canon" game. 

Such as...
- A 'rogue planet' appearing as a new system - various factions want control of the world when they find it once was home to an advanced civilisation whose technology might be valuable. But perhaps that civilisation isn't as extinct as everyone assumed? 
-Would probably work best as a plugin, but I thought it might be a different way to introduce a new faction, even if it's only a minor one. Potential to introduce new outfits, ships etc as well, none of which I'd be able to do myself.

- "Interspecies diplomacy" - It turns out that your crew are less tight lipped than yourself, and have been sharing stories of alien squirrels, bugs, and 'Giraffe people' at every bar, pub, vapour den, and inedbriation station from 14 Pole to W-3197. These rumours have found their way to various respective governments, all of whom want to get as much intelligence on the various factions of the Milky Way as possible. 
And you just happen to have the means to get them in a room together. Since these are all high level diplomats they'll obviously want to travel in style with their extensive entourage, so you'll have to be outfitted accordingly. Have you ever wished there was a good reason to build a fleet of luxury space liners guraded by a tough-as-nails escort of heavy warships ready to defend against professional assassins, rogue Quarg, and those pirates you always bump into who insist on attacking you when it's obviously suicidal? 
Well now there is because of what I said at the start of this bullet point.

- Neutron Star harvest - Mining is hardly the most exciting activity a swashbuckling, alien-with-weird-face-punching, financially prudent, Super-Cool Space Captain can engage in - but what about coralling a fragment of a neutron star? Thought this might be a way to make asteroid mining more lucractive and/or challenging in late game. I was thinking of a 'random' mission appearance where a chunk of a neutron star is passing through a nearby system. 
Destroying the chunk of neutron star would require a fleet with a very high 'asteroid damage' rating, the chunk would be moving at extremely high speed, and each chunk of neutron star would take up x 100 (or more) cargo space. If credits aren't much of a lure in the late game, I wondered if each chunk could be used to make a useful outfit/hand-to-hand item?

Really keen to hear everyone's feedback, critique, suggestions etc

For anyone who made it this far, thanks! And, if it lends my rambling a shred of credibility, I'm sort of 'published' as a sci fi writer, with a short story I wrote, variously called "Sanctity" or "Apeirophobia".

Cheers -VR

**Comment by eebop** [comment]
These are all very technically feasible, and number 1 sounds quite promising, actually

**Reply by Vemenous-Repentile** [reply]
Thanks! I could have a go at writing up a plot outline for potential missions if there's any interest

**Comment by Zitchas** [comment]
Before you go writing yourself off, I'll just point out that all the content for ES is plain text, and it's fairly easy to read, and modify them.

<details>

<summary>A typical mission (taken from Pookie Part 1, with text omitted)</summary>

```
mission "Transport Big Stuff"
	minor
	name "Mission Name the player will see here"
	description "Travel to <destination> and deliver the goods."
	source "Hestia"
	destination "Shroud"
	passengers 1
	cargo "Big Stuff" 15
	on offer
		conversation
			`The introductory paragraph. insert lots of text here.`
			`	A second paragraph (note the second and onwards paragraphs are indented). This paragraph asks the player if they will accept the mission or not, leading to the choice.`
			choice
				"	(Accept.)"
				"	(Decline.)"
					decline
			"	If they picked the decline option, the whole thing ends right there. If they don't, they'll get this text here, since it's the next thing after the choice. Choices can also send the player to specific points to build more intricate conversations and stories.."
				accept
```
</details>

It's a basic mission, but it should work (or get very close, anyway.) There's important reasons for the various things (like the number of tabs is very important. Many a mission has not worked because of a missed tab) But that's all there is to it to get started. The wiki has more detail on complexity. 

Ideas and concepts are good. Very good. Very useful. What's even more useful is people putting them into missions. All you need is a text editor.
